api/src/test/java/com/epam/pipeline/manager/pipeline/PipelineConfigurationManagerTest.java

```diff
@@ -16,182 +16,124 @@
 
-    private DockerRegistry registry;
-    private ToolGroup library;
-    private Tool tool;
-    private List<AbstractDataStorage> dataStorages = new ArrayList<>();
+    private final List<AbstractDataStorage> dataStorages = new ArrayList<>();
 
     @Before
     public void setUp() throws Exception {
-        registry = new DockerRegistry();
-        registry.setPath(TEST_REPO);
-        registry.setOwner(TEST_USER);
-        dockerRegistryDao.createDockerRegistry(registry);
-
-        library = new ToolGroup();
-        library.setName(TOOL_GROUP_NAME);
-        library.setRegistryId(registry.getId());
-        library.setOwner(TEST_USER);
-        toolGroupDao.createToolGroup(library);
-
-        tool = new Tool();
-        tool.setImage(TEST_IMAGE);
-        tool.setRam(TEST_RAM);
-        tool.setCpu(TEST_CPU);
-        tool.setOwner(TEST_USER);
-
-        tool.setRegistryId(registry.getId());
-        tool.setToolGroupId(library.getId());
-        toolDao.createTool(tool);
-
-        // Data storages of user 1
-        NFSDataStorage dataStorage = new NFSDataStorage(dataStorageDao.createDataStorageId(), "testNFS",
-                                                        "test/path1");
-        dataStorage.setMountOptions("testMountOptions1");
-        dataStorage.setMountPoint("/some/other/path");
-        dataStorage.setOwner(TEST_OWNER1);
-        dataStorageDao.createDataStorage(dataStorage);
-        dataStorages.add(dataStorage);
-
-        S3bucketDataStorage bucketDataStorage = new S3bucketDataStorage(dataStorageDao.createDataStorageId(),
-                                                                        "testBucket", "test/path2");
-        bucketDataStorage.setOwner(TEST_OWNER1);
-        dataStorageDao.createDataStorage(bucketDataStorage);
-        dataStorages.add(bucketDataStorage);
-
-        // Data storages of user 2
-        dataStorage = new NFSDataStorage(dataStorageDao.createDataStorageId(), "testNFS2", "test/path3");
-        dataStorage.setMountOptions("testMountOptions2");
-        dataStorage.setOwner(TEST_OWNER2);
-        dataStorageDao.createDataStorage(dataStorage);
-        dataStorages.add(dataStorage);
-
-        bucketDataStorage = new S3bucketDataStorage(dataStorageDao.createDataStorageId(),
-                                                                        "testBucket2", "test/path4");
-        bucketDataStorage.setOwner(TEST_OWNER2);
-        dataStorageDao.createDataStorage(bucketDataStorage);
-        dataStorages.add(bucketDataStorage);
-
-        dataStorages.forEach(ds -> aclTestDao.createAclForObject(ds));
-        aclTestDao.grantPermissions(dataStorage, TEST_OWNER1,
-                                    Collections.singletonList((AclPermission) AclPermission.READ));
+        MockitoAnnotations.initMocks(this);
+        dataStorages.add(getNfsDataStorage(NFS_ID_1, "test/path1", "options1", "/some/path", OWNER1));
```

Comment:
I would prefer to move such string to test class constants. It would be more accurate to test like:
```
private static final String TEST_MOUNT_POINT = "/some/path";
...
assertThat(mountPoints)
                .hasSize(1)
                .contains(TEST_MOUNT_POINT);
```
or
```
private static final String TEST_PATH1 = "test/path1"
...
assertThat(buckets)
                .hasSize(4)
                .contains(NFS_MASK + TEST_PATH1, ...);
```
Also, `NFS_MASK` can be initialized in  `DatastorageCreatorUtils`.

Reply:
Done!